querybook/webapp/components/DataTableView/DataTableHeader.tsx

```diff
@@ -198,7 +236,9 @@ export const DataTableHeader: React.FunctionComponent<IDataTableHeader> = ({
         <div className="DataTableHeader flex-column mb16">
             {topDOM}
             {titleDOM}
-            {ownerDOM}
+            {metastore.config[MetadataType.OWNER] !== MetadataMode.WRITE_LOCAL
```

Comment:
how does add ownership works now with ownership type?

Reply:
ownership type only be supported when syncing from metastore for now. If metatore is not configured to load owners, the behavior will remain the same as today.